.prettierrc.json

```diff
@@ -4,5 +4,6 @@
   "singleQuote": true,
   "printWidth": 120,
   "tabWidth": 2,
-  "endOfLine": "lf"
+  "endOfLine": "lf",
+  "plugins": ["prettier-plugin-tailwindcss"]
```

Comment:
web/ and docs/ refer to the top-level config currently. See also #4628 

Maybe it can be done in a different way, then one does not need to install this in their global npm package stack.

Reply:
Ah forget what I said, I think your proposed change should be just fine.